base/src/main/java/butter/droid/base/torrent/TorrentService.java

```diff
@@ -112,6 +109,7 @@ public void onDestroy() {
         }
 
         foregroundManager.setListener(null);
+        stopStreaming();
```

Comment:
I suppose we should stop the streaming when this service dies?

Reply:
I suppose :) This class I up for a major rewrite, so I wouldn't spend to much time on it.